homeassistant/components/spotify/__init__.py

```diff
@@ -30,7 +30,11 @@
 from . import config_flow
 from .browse_media import async_browse_media
 from .const import DOMAIN, LOGGER, SPOTIFY_SCOPES
-from .util import is_spotify_media_type, resolve_spotify_media_type
+from .util import (
+    is_spotify_media_type,
+    resolve_spotify_media_type,
+    spotify_uri_from_media_browser_url,
```

Comment:
This needs to be used in the media player play_media command too?

Reply:
When browsing with the Spotify media player itself, there's no config entry id added, it's only needed for users of `spotify.async_browse_media`.
In the current codebase, only sonos needs to be updated.